runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1346,8 +1346,9 @@ NNFW_STATUS nnfw_session::train_prepare()
 
   try
   {
-    if (not _train_info->isValid())
-      throw std::runtime_error{"training info is not valid"};
+    const auto res = _train_info->isValid();
+    if (not res.valid)
+      throw std::runtime_error{"training info is not valid \n" + res.error_msg};
```

Comment:
After this PR, `train_prepare` show that - why training info is invalid. 

### Before this PR 

When the user sets invalid training info, 
`train_prepare`fails without showing which training information is invalid.

```
./onert_train --modelfile mnist_with_meta.circle --load_input:raw pool_mnist_model/mnist_x_train.1000.bin --load_expected:raw pool_mnist_model/mnist_y_train.1000.bin --epoch 10  --learning_rate=0 --batch_size=0

Model Expected Filename pool_mnist_model/mnist_y_train.1000.bin
Model Input Filename pool_mnist_model/mnist_x_train.1000.bin
Model Filename mnist_with_meta.circle
== training parameter ==
- learning_rate   = 0
- batch_size      = 0
- loss_info       = {loss = mean squared error, reduction = use automatic reduction type}
- optimizer       = sgd
========================
Error during nnfw_session::train_prepare : training info is not valid 
```

### After this PR 

`train_prepare` shows the message like 'batch size must not be zero', 'learning rate must be positive'. 

```bash 
./onert_train --modelfile mnist_with_meta.circle --load_input:raw pool_mnist_model/mnist_x_train.1000.bin --load_expected:raw pool_mnist_model/mnist_y_train.1000.bin --epoch 10  --learning_rate=0 --batch_size=0

Model Expected Filename pool_mnist_model/mnist_y_train.1000.bin
Model Input Filename pool_mnist_model/mnist_x_train.1000.bin
Model Filename mnist_with_meta.circle
== training parameter ==
- learning_rate   = 0
- batch_size      = 0
- loss_info       = {loss = mean squared error, reduction = use automatic reduction type}
- optimizer       = sgd
========================
Error during nnfw_session::train_prepare : training info is not valid 
batch size must not be zero
learning rate must be positive

```